plugins/node/opentelemetry-instrumentation-pg/src/instrumentation.ts

```diff
@@ -435,6 +437,52 @@ export class PgInstrumentation extends InstrumentationBase<PgInstrumentationConf
     };
   }
 
+  private setPoolConnectEventListeners(pgPool: PgPoolExtended) {
```

Comment:
This should probably be prefixed with an underscore to match the other private methods.

Reply:
done